e2e/test/scenarios/dashboard-filters/dashboard-filter-defaults.cy.spec.ts

```diff
@@ -125,6 +128,9 @@ describe("scenarios > dashboard > filters > reset", () => {
     cy.location("search").should("eq", "?filter_one=Quu&filter_two=Foo");
     filterWidget().contains("Quu").should("be.visible");
     filterWidget().contains("Foo").should("be.visible");
+
+    clearFilterWidget(0);
+    clearFilterWidget(1);
```

Comment:
This is needed because the user's last used values for the filters are not reset by `restore` and this causes issues when running the same test consecutively in the stress test using `--burn`.